quiz-game/src/lib/audioManager.svelte.ts

```
@@ -0,0 +1,222 @@
+        }
+    }
+
+    /**
+     * Initialize global window listeners to unlock audio
+     */
+    init() {
+        if (!browser) return;
+        const handler = () => {
+            this.unlockAudio();
+            if (this.hasUnlocked) {
+                ['click', 'keydown', 'touchstart'].forEach(e => window.removeEventListener(e, handler));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)